src/api/libcellml/annotator.h

```diff
@@ -89,23 +89,23 @@ class LIBCELLML_EXPORT Annotator: public Logger
      *  - <CellmlElementType::RESET_VALUE, @ref ResetPtr> - Retrieve the identifier with Reset::resetValueId (const std::string &).
      *  - <CellmlElementType::TEST_VALUE, @ref ResetPtr> - Retrieve the identifier with Reset::testValueId (const std::string &).
      *  - <CellmlElementType::UNDEFINED, @c nullptr>.
-     *  - <CellmlElementType::UNIT, @ref UnitPtr> - Retrieve the identifier with Units::unitId(size_t) const.
+     *  - <CellmlElementType::UNIT, @ref UnitsItemPtr> - Retrieve the identifier with Units::unitId(size_t) const.
      *  - <CellmlElementType::UNITS, @ref UnitsPtr> - Retrieve the identifier with Units::id().
      *  - <CellmlElementType::VARIABLE, @ref VariablePtr> - Retrieve the identifier with Variable::id().
      *
      * @param id A @c std::string representing the @p id to retrieve.
      *
-     * @return An @c AnyItem item (as described above).
+     * @return An @ref AnyCellmlElement item (as described above).
      */
```

Comment:
I don't think this documentation is accurate any more.

Reply:
Done in commit c28817c1.